src/controllers/restaurantsController.ts

```diff
@@ -11,16 +11,20 @@ import {
   getRestaurantById,
   getRestaurantMenuByRestaurantId,
 } from "../data/restaurant.js";
-
 import restaurantsNearYouSchema from "../schemas/restaurantsNearYouReq.js";
 import {
   RestaurantOverviewReqSchema,
   RestaurantOverviewResSchema,
 } from "../schemas/schemas.js";
+import restaurantOverviewQueryParamsSchema from "../schemas/restaurantOverviewQueryParams.js";
+import convertIntoNumbers from "../helpers/convertIntoNumers.js";
 
 const defaultRange = Number(process.env.DEFAULT_RANGE);
 const defaultLatitude = Number(process.env.DEFAULT_LAT);
 const defaultLongitude = Number(process.env.DEFAULT_LON);
+const rangeQueryName = process.env.RANGE_QUERY_NAME;
+const userLongitudeQueryName = process.env.USER_LONGITUDE_QUERY_NAME;
+const userLatitudeQueryName = process.env.USER_LATITUDE_QUERY_NAME;
```

Comment:
This feels completely unnecessary. As far as I can tell it is only used as a key for a temporary map. It could be named anything, adding extra environmental variables are not needed here.